spinoso-time/src/time/tzrs/convert.rs

```diff
@@ -24,38 +23,73 @@ impl Display for Time {
     /// [`Time#ctime`]: https://ruby-doc.org/core-3.1.2/Time.html#method-i-ctime
     /// [`Time#to_s`]: https://ruby-doc.org/core-3.1.2/Time.html#method-i-to_s
```

Comment:
I have a feeling this can be cut down to just 

```rust
    /// [`Time#to_s`]: https://ruby-doc.org/core-3.1.2/Time.html#method-i-to_s
```

Reply:
yes, forgot to fix this up when reworking the docs.